### DISCUSSION TITLE
Nazwy parametrów w komunikatach błędów

### DISCUSSION THREAD

**Discussion by Onixarts** [allegro/allegro-api#5413] [Suggestions - resolved]
Prosiłbym o dodanie do treści błędów, które informują o parametrach specyficznych, nazwy tych parametrów. Błędy w `userMessage` wyświetlamy bezpośrednio użytkownikom a w samym interfejsie programu użytkownicy nie widzą identyfikatorów tylko nazwy i trudno zorientować się o jaki parametr chodzi.

Dotyczy to endpointów do dodawania/aktualizacji ofert jak i zgłaszania propozycji produktu.

```json
{
    "errors": [
        {
            "code": "EmptyCustomValueOnAmbiguousValueException",
            "message": "227561",
            "details": null,
            "path": null,
            "userMessage": "Nie została podana propozycja wartości do wartości niejednoznacznej w parametrze 227561"
        }
    ]
}
```

A mogło by to wyglądać tak:

```json
{
    "userMessage": "Nie została podana propozycja wartości do wartości niejednoznacznej w parametrze 'Zastosowanie' (227561)"
}
```

**Comment by MartaNowaczyk** [comment]
Zmiana ta jest już od dłuższego czasu wdrożona. Zmieniam status na "resolved".
